### ISSUE TITLE
Error: No lintable files found at paths: ''

### ISSUE THREAD

**Issue by ryansalisbury** [realm/SwiftLint#5850]
### New Issue Checklist

- [ ] I've Updated SwiftLint to the latest version.
- [ ] I've searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues).

### Bug Description

I have followed the installation instructions from the github readMe, and when I run swiftlint form the terminal, it correctly outputs the field with linting warnings/errors in them. However, when I add the build script to the Build Phase as 'SwiftLint'm using the bash code found in the ReadME, I consistently get this error:

Linting Swift files in current working directory
Error: No lintable files found at paths: ''
Command PhaseScriptExecution failed with a nonzero exit code

Mention the command or other SwiftLint integration method that caused the issue. Include stack traces or command output.

```bash
Linting Swift files in current working directory
Error: No lintable files found at paths: ''
Command PhaseScriptExecution failed with a nonzero exit code

```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)
* 0.57.0
* Xcode version (run `xcodebuild -version` to be sure)
* Xcode 16.1
* Build version 16B40
* Installation method used (Homebrew, CocoaPods, building from source, etc)
* I used HomeBrew to install swiftlint
* Configuration file:

```yml
I do not have a yml file set-up for configurations since I am just installing swiftlint for the first time.```

Are you using [nested configurations](https://github.com/realm/SwiftLint#nested-configurations)? If so, paste their
relative paths and respective contents.
I am not using nested configurations - i am just starting a swift project for the first time and am surprised that Swift does not have a plugin like 'Prettier' for VSCode.

**Comment by vdhamer** [comment]
Try setting Build Settings > Build Options > User Script Sandboxing to "No".
I am running Xcode 16.2 beta 3 (16C5023f) and can confirm the issue on a small test project.
This issue has been around for a while:
[https://github.com/realm/SwiftLint/issues/5053](https://github.com/realm/SwiftLint/issues/5053)

**Comment by vdhamer** [comment]
Arggggh. Couldn't figure out how to fix an error message. And Google gives me my own posting how to solve it.
